lib/membrane_rtc_engine/endpoints/webrtc_endpoint.ex

```diff
@@ -143,29 +121,39 @@ defmodule Membrane.RTC.Engine.Endpoint.WebRTC do
     state = %{
       extensions: opts.extension_options,
       packet_filters: opts.packet_filters || %{},
-      payload_and_depayload_tracks?: opts.payload_and_depayload_tracks?,
-      outbound_tracks: Map.new(opts.outbound_tracks, &{&1.id, &1})
+      outbound_tracks: %{},
+      inbound_tracks: %{}
     }
 
     {{:ok, spec: spec, log_metadata: opts.log_metadata}, state}
   end
 
   @impl true
   def handle_notification({:new_tracks, tracks}, _from, _ctx, state) do
```

Comment:
Not sure about that, handle_notification indicates that the message comes from children, while handle_other - from another place, so that's what marks the difference. We're also going to make it more clear, see https://github.com/membraneframework/membrane_core/issues/270